### PR TITLE
solutionPerson

### PR DESCRIPTION
**Pull request by NasyTheSlayer**
[DEMO LINK](https://NasyTheSlayer.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:2]: it's a good practice to use object destructing 
```suggestion
export const Person = ({ person }) => (
  const { name, age, ... } = person;
  
  return (
    <section className="Person">
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,18 @@
...
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:4]:
```suggestion
      My name is {name}
```

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,18 @@
...
+    </h2>
+    { person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:10]:
```suggestion
    {age && (
      <p className="Person__age">
        I am {person.age}
      </p>
    )}
```

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,18 @@
...
+    <p className="Person__partner">
+      { person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:13]:
```suggestion
      {isMarried
        ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/components/Person/Person.jsx [hunk 5]
```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:7]: you can simplify without using template string
```suggestion
        My name is {name}
```

File: src/components/Person/Person.jsx [hunk 6]
```diff
@@ -1 +1,22 @@
...
+      </h2>
+      { age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
```
Comment (Viktor-Kost) [inline src/components/Person/Person.jsx:13]: the same
```suggestion
      {age && (
        <p className="Person__age">
          I am {age}
        </p>
      )}
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Just minor fixes needed

**Review by nazarmatsevych — APPROVED** [review]
👍

**Pull request closed without merging**